.snyk

```diff
@@ -1,3 +1,12 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
```

Comment:
If we can remove expiry date, no need for this comment

Reply:
let me try ...  done: removed expiry and comment line